app/views/featured_pages/show.js.erb

```diff
@@ -0,0 +1,2 @@
+var html = "<%= j(render(partial: 'pages/featured_column', locals: {page: @page})) %>"
```

Comment:
alias for `escape_javascript` - http://api.rubyonrails.org/classes/ActionView/Helpers/JavaScriptHelper.html#method-i-j